docker-app/qfieldcloud/filestorage/view_helpers.py

```diff
@@ -233,6 +238,36 @@ def download_field_file(
     http_host = request.headers.get("host", "")
     https_port = http_host.split(":")[-1] if ":" in http_host else "443"
 
+    download_range = request.headers.get("Range", "")
+    if download_range:
+        range_match = parse_range(download_range)
+
+        if not range_match:
+            return HttpResponse(
+                "`Range` HTTP header must be formatted like `bytes=start-(end)`",
+                status=416,
+            )
+
+        file_size = field_file.size
+        range_start = range_match[0]
+
+        if range_match[1]:
+            range_end = range_match[1]
+        else:
+            range_end = file_size - 1
```

Comment:
```suggestion
        file_size = field_file.size
        range_start, range_end = range_match[0]

		if range_end is None:
		    range_end = file_size - 1
```

It is safer in this case to explicitly check for `None`.